.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,59 @@
+name: Build artifact
+
+on:
+  push:
+    paths:
+      - 'plugin/**'
```

Comment:
Can you also include these files in `paths`:
* `util/publish.sh`
* `.github/workflows/build-artifact.yml`
* `util/DEPS.py`
* `util/fetch_deps.py`

Reply:
Okay!